web/src/components/ToolbarOption.vue

```diff
@@ -0,0 +1,31 @@
+<script>
```

Comment:
This should maybe be called `RadioGroup` or `OptionGroup` or something like that.

Reply:
`RadioGroup` is too general imo. This one is designed to work precisely in the left toolbar of an analysis.

Reply:
I'll rename this.